src/modules/flow/oic/Makefile

```diff
@@ -1,3 +1,3 @@
 obj-$(FLOW_NODE_TYPE_OIC) += oic.mod
-obj-oic-$(FLOW_NODE_TYPE_OIC) := oic.json oic.o
+obj-oic-$(FLOW_NODE_TYPE_OIC) := core.light.json oic.json oic.o
```

Comment:
It was by mistake, sorry, it had to be git added back here. I'll go with a different entry there specific for OIC templates, for now, then. I'll maybe wait for you big-phat-build-system patch to get in before mine gets (let's see if I figure out the damn samples bug soon, also).